### DISCUSSION TITLE
build unsigned UTM without admin?

### DISCUSSION THREAD

**Discussion by TechDudie** [utmapp/UTM#6587] [UTM Help (macOS)] [answered]
Hi, I'm trying to get UTM working properly on my school macbook. I want to virtualize windows 11 on my macbook m1. However, theres a piece of software blocking UTM and other apps, autoterminating it when it is started. I tried to build it myself, but xcode requires admin to install. Is there a way to build UTM without codesigning through something like github actions?

**Comment by Hardwaregore** [comment]
Hi @TechDudie 

I see you’re trying to get UTM running on your school MacBook M1 to virtualize Windows 11. That sounds cool in all but you need to remember that school devices have restrictions for a reason. These measures are in place to ensure the devices are used for educational purposes and to maintain security.

Trying to bypass these restrictions or using unauthorized methods to install software can lead to unintended consequences and might violate your school’s policies. Instead, I suggest focusing on using the resources and tools that are already available to you for your studies.

Remember, ***school is a place to study***, and it’s important to respect the rules and resources provided.

**Reply by TechDudie** [reply]
Hi Mr. <wait I forgot that I can't dox you even though you I know you irl and you did that to me also.. I'll just stick with> Mr. Academic Knowitall, please mind your own business over in your sillier school district. I choose what to do with my school device, and I am responsible for the consequences either way.

**Reply by TechDudie** [reply, marked as answer]
Anyhow, it's quite simple. Just spin up a QEMU VM using technomagic, boot into macOS, install Xcode and build on there (unsigned), transfer it to NFTS or FAT32 to get rid of APFS's xtended attrs, and there you go. Goodbye, Mr Certified Bad-at-networking, and I hope I never see you irl ever again. 😠 

p.s. I swear you used ChatGPT to generate your yap because you didn't misspeel any words.
